src/transforms/replace.jl

```diff
@@ -0,0 +1,30 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Replace(old₁ => new₁, old₂ => new₂, ..., oldₙ => newₙ)
+
+Replaces all occurrences of `oldᵢ` with `newᵢ` in the table.
```

Comment:
```suggestion
Replaces `oldᵢ` value with `newᵢ` value in the table.
```

Reply:
OK

Reply:
Made in [6e3d31e](https://github.com/JuliaML/TableTransforms.jl/pull/42/commits/6e3d31efa6523dd7b8c4fb85c21d07ac9cc4e738)